seqr/views/utils/individual_utils.py

```diff
@@ -52,7 +52,7 @@ def add_or_update_individuals_and_families(project, individual_records, user, ge
     families_by_id = {f.family_id: f for f in Family.objects.filter(project=project, family_id__in=family_ids)}
 
     missing_family_ids = family_ids - set(families_by_id.keys())
-    for family_id in missing_family_ids:
+    for family_id in sorted(missing_family_ids):
```

Comment:
Is this a flappy test fix?

Reply:
this PR would have resulted in a flappy test if not for this change. Previously, we sorted the sample ids for export in anvil loading. Now, the way I modified the code to share behavior for exporting the sample ids in elasticsearch or the pedigree file on the hail backend, both exports are sorted the same way which is by family db id and then by sample id. So for a test that adds multiple families, the sort order by just sample id was deterministic but the sort by family and sample id was not if the families were not created in a reliable order